CodeGen/src/IrLoweringX64.cpp

```diff
@@ -619,13 +620,13 @@ void IrLoweringX64::lowerInst(IrInst& inst, uint32_t index, const IrBlock& next)
     {
         inst.regX64 = regs.allocRegOrReuse(SizeX64::xmmword, index, {inst.a, inst.b});
 
-        ScopedRegX64 tmp1{regs, SizeX64::xmmword};
-        ScopedRegX64 tmp2{regs, SizeX64::xmmword};
+        ScopedRegX64 tmp1{regs};
+        ScopedRegX64 tmp2{regs};
 
-        // Fourth component is the tag number which is interpreted as a denormal and has to be filtered out
-        build.vandps(tmp1.reg, regOp(inst.a), vectorAndMaskOp());
-        build.vandps(tmp2.reg, regOp(inst.b), vectorAndMaskOp());
-        build.vsubps(inst.regX64, tmp1.reg, tmp2.reg);
+        RegisterX64 tmpa = vecOp(inst.a, tmp1);
+        RegisterX64 tmpb = vecOp(inst.b, tmp2);
```

Comment:
Can you add a comment describing why sub/div exclude the `inst.a == inst.b` check.
Because I have no idea.

Reply:
I can add handling for this, it just felt redundant because why are we accelerating `a/a` or `a-a` (whereas `a*a` and `a+a` occur in practice).